src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, op) => {
+  const numOne = Big(Number(numberOne));
+  const numTwo = Big(Number(numberTwo));
+  let result = new Big();
+
+  switch (op) {
+    case '+':
+      result = numOne + numTwo;
+      break;
+    case '-':
+      result = numOne - numTwo;
+      break;
+    case '*':
+      result = numOne * numTwo;
+      break;
+    case '÷':
+      result = numOne / numTwo;
+      break;
+    case '%':
+      result = numOne % numTwo;
+      break;
```

Comment:
`%` should turn the number into a percentage based number, `e.g. 95 % == .95` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))